homeassistant/components/homeassistant/triggers/event.py

```diff
@@ -19,36 +20,42 @@
         vol.Required(CONF_PLATFORM): "event",
         vol.Required(CONF_EVENT_TYPE): cv.string,
         vol.Optional(CONF_EVENT_DATA): dict,
+        vol.Optional(CONF_EVENT_CONTEXT): dict,
     }
 )
 
 
+def _populate_schema(config, config_parameter):
+    if config_parameter not in config:
```

Comment:
The existing behavior was changed slightly with this check. The old check only set a event data schema if the config had a populated event data dict. The current check sets a schema as long as the event data key is in the config. It doesn't check for a truthy event data dict.

Reply:
Good catch. I'm not sure this makes any practical sense (since the schema will be empty), but I'll fix this together with the multiple user selection below.

Reply:
https://github.com/home-assistant/core/pull/41036/files#diff-aa600ac67a083c2d308272918338761fR36-R37